### ISSUE TITLE
'Show Playlist' Always Appears With Several Videos In Directory

### ISSUE THREAD

**Issue by Geyup** [u8sand/Baka-MPlayer#212]
Hello,

I compiled 2.0.4 from source, below are my system specs;

Slackware 14.1 x86_64
Openbox 3.6.1
tint2 0.12.11
QT5 5.5.1

I noticed that if I have several videos in the same directory, and I pick one, when it starts to play, 'Show Playlist' appears, and is checked in the menu, and all the videos in that directory appear in the 'Show Playlist'

I'm assuming for the moment, the playlist is only to show the video you are playing? Also if it's unchecked and when restarting Baka-MPlayer it won't appear, but in this situation it does.

Hmm

**Comment by godly-devotion** [comment]
That is the intended behavior. If a file was opened and there are other playable files in the same folder, Baka will display the playlist to show that it has automatically populated the playlist. Conversely, if there are no other files in the same folder, the playlist will hide (but still viewable).

**Comment by Geyup** [comment]
Hmm to be honest this is the first app I've ever seen act this way. Maybe there are others that take this approach. I've always felt like this is a behviour left to the end-user to decide how it's going to open and dispaly, or not be displayed.

I think this would be good to create a preference for this, for the end-user to pick how they want the playlist to behave, or be viewed, I personally don't care or use a playlist, so I want it hidden and never shown...

thanks

**Comment by viliampucik** [comment]
I agree with @Geyup. It would be nice if say disabled `Settings -> Show Playlist` option would be preserved across the player restarts and no playlist would be show even for directories containing  playable files.

**Comment by godly-devotion** [comment]
As mentioned in issue #220, the option to disable the playlist from automatically showing up will be added in the next incarnation.

**Comment by viliampucik** [comment]
I see now. Thanks @godly-devotion :)

**Comment by Soitora** [comment]
It has gone over two years now, when is this getting added? I start Baka MPlayer with "Hide All Controls", but it launches with playlist-then hides it, which adds these large ugly black bars to my screen. 

I don't use the playlist feature at all and I don't wanna have to switch from my favorite video player because a simple visibility toggle never was added.
